### PR TITLE
Apply formatter to spotbugs/test, spotbugs/xml, spotbugs/workflow, spotbugs/visitclass, spotbugs/util

### PR DESCRIPTION
**Pull request by wreulicke**
Apply formatter below.

* spotbugs/testjava
* spotbugs/main/java/edu/umd/cs/findbugs/xml
* spotbugs/main/java/edu/umd/cs/findbugs/workflow
* spotbugs/main/java/edu/umd/cs/findbugs/visitclass
* spotbugs/main/java/edu/umd/cs/findbugs/util

This PR is cherry-picked from https://github.com/wreulicke/spotbugs/tree/feature/apply-spotless

### CODE REVIEW

File: spotbugs/src/main/java/edu/umd/cs/findbugs/util/Util.java [hunk 1]
```diff
@@ -338,7 +341,7 @@ public static String getFileExtensionIgnoringGz(File f) {
 
     public static boolean isPowerOfTwo(int i) {
         return i > 0
-            && (i | (i - 1)) + 1 == 2 * i;
```
Comment (ThrawnCA) [inline spotbugs/src/main/java/edu/umd/cs/findbugs/util/Util.java]: I don't really agree with the extra indentation here, but eh, if it means we can apply consistent formatting, it's a win.

File: spotbugs/src/main/java/edu/umd/cs/findbugs/visitclass/AnnotationVisitor.java [hunk 2]
```diff
@@ -143,8 +143,8 @@ public void visitSyntheticParameterAnnotation(int p, boolean runtimeVisible) {
         }
         return values;
     }
-
-
+    
+
```
Comment (ThrawnCA) [inline spotbugs/src/main/java/edu/umd/cs/findbugs/visitclass/AnnotationVisitor.java]: Should we just delete a line here?

Comment (wreulicke) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/visitclass/AnnotationVisitor.java]: I do not think so. It is fine like this.
IMO, To remove comment-out lines is better than to delete a line.

I would like to separate issue.

File: spotbugs/src/main/java/edu/umd/cs/findbugs/util/Util.java (hunk 1 above)
Comment (wreulicke) [reply spotbugs/src/main/java/edu/umd/cs/findbugs/util/Util.java]: > I don't really agree with the extra indentation here
I think so, but i seem that extra indentations is contained in code base.
so, I will merge for consistent formatting.

**Pull request merged**